module/os/windows/spl/spl-kmem.c

```diff
@@ -3549,7 +3549,7 @@ kmem_cache_create(
 	/*
 	 * Set cache properties.
 	 */
-	(void) strncpy(cp->cache_name, name, KMEM_CACHE_NAMELEN);
+	(void) strlcpy(cp->cache_name, name, KMEM_CACHE_NAMELEN + 1);
```

Comment:
The call bellow passes KMEM_CACHE_NAMELEN + 1 so I'm assuming that's the actual size of the buffer

Reply:
They change the name field from `KMEM_CACHE_NAMELEN + 1` to just `KMEM_CACHE_NAMELEN` in the header file, then add +1 in the `strlcpy()`. So it works, as long as the header is updated too

Reply:
did they?
i only saw changes to KSTAT_STRLEN